lib/cc/cli/output.rb

```diff
@@ -0,0 +1,34 @@
+module CC
+  module CLI
+    module Output
+      def success(message)
+        terminal.say colorize(message, :green)
+      end
+
+      def say(message)
+        terminal.say message
+      end
+
+      def warn(message)
+        terminal.say(colorize("WARNING: #{message}", :yellow))
```

Comment:
:lipstick: I realize this was existing, but can you drop the outer parens so all invocations of `#say` are consistent here?